index.js

```diff
@@ -0,0 +1,127 @@
+require('dotenv').config();
+const express = require("express");
+// const helmet = require("helmet");
+const bcrypt = require("bcryptjs");
+const dbFuncs = require("./dbFunctions");
+const jwt = require("jsonwebtoken");
+const morgan = require("morgan");
+// const KnexSessionStore = require("connect-session-store")(session);
+const server = express();
+
+// server.use(helmet());
+server.use(express.json());
+server.use(morgan())
+
+
+server.get("/", (req, res) => {
+  res.status(200).send("<h1>The server is up</h1>");
+});
+
+server.post("/users/register", async (req, res) => {
+  const userInfo = req.body;
+  const hash = bcrypt.hashSync(userInfo.password, 12);
+  userInfo.password = hash;
+
+  try {
+    const response = await dbFuncs.addUser(userInfo);
+    res.status(201).json(response);
+  } catch (err) {
+    console.log(err);
+    res.status(500).json(err);
+  }
+});
+
+function generateToken(user) {
```

Comment:
Pretty sure if you wrapped `user` in curlys like `function generateToken({username, name}) {`, then on line 36 you could just do `username: username`. Same thing on `name: name`. (Then you could destructure that to just ` { username, name}`